pumpkin-protocol/src/client/config/c_store_cookie.rs

```diff
@@ -0,0 +1,23 @@
+use pumpkin_macros::client_packet;
+
+use crate::{Identifier, VarInt};
+
+#[derive(serde::Serialize)]
+#[client_packet("config:store_cookie")]
+/// Stores some arbitrary data on the client, which persists between server transfers.
+/// The Notchian client only accepts cookies of up to 5 kiB in size.

```

Comment:
```suggestion
/// The Notchian (vanilla) client only accepts cookies of up to 5 kiB in size.
```
Most people should understand what the "Notchian" client means, but... Why not?